src/uuid.cr

```diff
@@ -202,6 +202,16 @@ struct UUID
     @bytes.to_slice.hexstring
   end
 
+  # Returns a `String` that is a valid urn of *self*
+  #
+  # ```
+  # require "uuid"
+  #
+  # uuid = UUID.empty
+  # uuid.urn # => "urn:uuid:00000000-0000-4000-0000-000000000000"
+  # uuid2 = UUID.random
```

Comment:
It would be better to use a fixed value here in order to ensure reproducibility of the example.

Reply:
good idea.  soved by https://github.com/crystal-lang/crystal/pull/11693/commits/80420c0421ff7f28622d2db024e914ab4729fa76